ui/shared/CopyToClipBoardButton.qml

```diff
@@ -44,6 +47,31 @@ Rectangle {
             onClick()
         }
     }
+
+    ToolTip {
+        id: toolTip
+        text: parent.textTooltip
+        parent: copyToClipboardButton
+    }
+
+    Timer {
+        id:showTimer
+        interval: 500
+        running: parent.showTooltip && !toolTip.visible
+        onTriggered: {
+            toolTip.visible = true;
```

Comment:
I think you can set the tooltip to `visible` straight away and not need the 500 timer. Unless you tell me it was too ugly?

Reply:
Generally, tooltip has a little delay before it is shown.
If you think 0.5 sec is too long, let me update more short delay like 0.2 or 0.3